Update README.md (#14)
agustinaGallo97/TaskLoans-Android: README.md

# Taskloans project

This is [Xmartlabs](https://xmartlabs.com) Android Template project.
Please read through this README before start working.

# Taskloans description

> TaskLoans is a project that consists of managing the washing of dishes in a company

Reviewer: ```suggestion
TaskLoans is a project that helps users in a specific company to keep track of dishwashing chores.
```

## Style guide

Please, remember to always use and follow our [style guide](https://github.com/xmartlabs/Android-Style-Guide)
when making contributions to this project.

> ## :link: Jira Tracker reference:

Reviewer: ```suggestion
## :link: Jira reference:
```

> https://androidtraining.atlassian.net/browse/AT-107

Reviewer: ```suggestion
https://androidtraining.atlassian.net
```

> # Android Template project

Reviewer: taskloans

> This is [Xmartlabs](https://xmartlabs.com) Android Template project.

Reviewer: taskloans description
